beta-20161110.md

```diff
@@ -0,0 +1,76 @@
+---
+title: What's New in beta-20161110
+toc: false
+summary: Additions and changes in CockroachDB version beta-20161110.
+---
+
+## Nov 10, 2016
+
+### Binaries
+
+<div id="os-tabs" class="clearfix">
+    <a href="https://binaries.cockroachdb.com/cockroach-beta-20161110.darwin-10.9-amd64.tgz"><button id="mac" data-eventcategory="mac-binary-release-notes">Mac</button></a>
+    <a href="https://binaries.cockroachdb.com/cockroach-beta-20161110.linux-amd64.tgz"><button id="linux" data-eventcategory="linux-binary-release-notes">Linux</button></a>
+</div>
+
+### Upgrade Notes
```

Comment:
I think titling this section "Backwards-Incompatible Change", as we've done in the past, would make this stand out more. But then we might want to write a sentence or two about the nature of this change. What do you think, @bdarnell?